src/main/java/com/coinbase/exchange/api/payments/PaymentService.java

```diff
@@ -1,29 +1,32 @@
 package com.coinbase.exchange.api.payments;
 
 import com.coinbase.exchange.api.exchange.GdaxExchange;
+import java.util.List;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.core.ParameterizedTypeReference;
 import org.springframework.stereotype.Component;
-
-import java.util.List;
+import org.springframework.stereotype.Service;
 
 /**
  * Created by robevansuk on 16/02/2017.
  */
 @Component
+@Service
```

Comment:
seems `@Service` is a specialised @Component - don't need both. Pls correct me if wrong.